### DISCUSSION TITLE
Help Needed: Lost Access to GitHub Account Due to 2FA Issues

### DISCUSSION THREAD

**Discussion by hichgram** [community/community#147902] [Other Feature Feedback, Questions, & Ideas]
Dear GitHub Community,

As the title suggests, I’ve lost access to my main GitHub account due to 2FA issues. Here’s what happened:

- My phone had issues, and I had to perform a factory reset.
- I used Microsoft Authenticator for GitHub 2FA, and although I had cloud backup enabled, I mistakenly logged into the app on my new phone instead of using the recovery option. This overwrote the previous backup with a new one, and now I can only restore the latest (empty) backup.
- Unfortunately, I didn’t save my GitHub recovery codes.
- I’ve tried every available recovery option but haven’t been able to regain access. I also cannot contact GitHub Support directly without logging in.

Here’s what I do have:

- Access to the primary email associated with the account.
- My username, which is my full name.
- Access to all commit messages, repositories, and emails linked to my account.
- Any documentation or proof of ownership that might be required.
- If anyone knows how to access older backups in Microsoft Authenticator or has advice on how I can regain access to my GitHub account, I would greatly appreciate it.

Thank you for your help!

**Comment by shinybrightstar** [comment]
Hi @hichgram , 👋

If you’ve lost access to your two-factor authentication (2FA) credentials, you can regain access to your account by using your recovery codes or another recovery option. Please visit GitHub’s[ Virtual Assistant](https://support.github.com/contact/account?contact%5Btype%5D=account&flow=2fa) to explore your available options.

If GitHub doesn’t support 2FA via text message in your country, you’ll need to set up authentication using a TOTP application. For more information on configuring 2FA with a TOTP app, here is the [documentation](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication-using-a-totp-mobile-app).

Please visit our[ FAQ](https://github.com/orgs/community/discussions/68240) or [explore similar discussions](TBD) in case you have additional questions.

If you’re unable to resolve your issue with the resources above, please open a ticket in our[ Support Portal](https://support.github.com/contact/account), as no other Community Discussions team members will be able to assist with account-related questions.
We appreciate your understanding!

**Reply by hichgram** [reply]
Hi @shinybrightstar,
Thanks for you answer, the problem is that none of the recovery methods work for me,
I opened a ticked when you told me, but still after more than a month, I haven't got any answer yet!
